src/ServiceComposer.AspNetCore/ServiceComposer.AspNetCore.csproj

```
@@ -34,7 +34,7 @@
 
   <Target Name="AppVeyorPullRequestsTarget" AfterTargets="MinVer" Condition="'$(APPVEYOR_PULL_REQUEST_NUMBER)' != ''">
     <PropertyGroup>
-      <PackageVersion>$(MinVerMajor).$(MinVerMinor).$(MinVerPatch)-PR.$(APPVEYOR_PULL_REQUEST_NUMBER).$(MinVerPreRelease)</PackageVersion>
+      <PackageVersion>$(MinVerMajor).$(MinVerMinor).$(MinVerPatch)-PR.$(APPVEYOR_PULL_REQUEST_NUMBER)-Build.$(APPVEYOR_BUILD_NUMBER)-$(MinVerPreRelease)</PackageVersion>
```

Bear in mind that the dash means that, according to SemVer, the pull request number plus the dash plus "Build" will form one of the single dot-separated identifiers.

If you want the build number to be called out explicitly, I would recommed this format (with consistent case thrown in by my OCD 😉):

```suggestion
      <PackageVersion>$(MinVerMajor).$(MinVerMinor).$(MinVerPatch)-pr.$(APPVEYOR_PULL_REQUEST_NUMBER).build.$(APPVEYOR_BUILD_NUMBER)-$(MinVerPreRelease)</PackageVersion>
```

---

Or alternatively, you could have the PR and build labels and their numbers as individual identifiers:

```suggestion
      <PackageVersion>$(MinVerMajor).$(MinVerMinor).$(MinVerPatch)-pr-$(APPVEYOR_PULL_REQUEST_NUMBER).build-$(APPVEYOR_BUILD_NUMBER).$(MinVerPreRelease)</PackageVersion>
```

---

Thanks. I like the first one more. I’ll take that.

---

Interestingly I cannot commit suggestions. I’ll close this one and commit directly to the original one your suggestion. Thanks.